ports/formula_1_playdate/README.md

```diff
@@ -0,0 +1,16 @@
+## Notes
+<br/>
+
+Thanks to [Willems Davy aka joyrider3774](https://github.com/joyrider3774) for this game and making available for free. Also thanks to Willems Davy aka Joyrider3774 for the packaging for portmaster.
```

Comment:
LOL

Reply:
yeah was not sure what todo there lol :)